### PR TITLE
Features/dbcopy group host display

### PR DESCRIPTION
**Pull request by marcoooo**
Fixed display for host / group restriction.

### CODE REVIEW

File: ensembl_production/templates/inc/headers.inc.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
   <!-- Bootstrap core CSS -->
 
     <link href="{% static "css/bootstrap.min.css"%}" rel="stylesheet"/>
-    <link href="{% static "css/bootstrap-reboot.min.css"%}" rel="stylesheet"/>
+    <link href="{% static "css/bootstrap-reboot.min.css"%}" rel="stylesheet">
```
Comment (james-monkeyshines) [inline ensembl_production/templates/inc/headers.inc.html:11]: I think the trailing slash be retained?

File: production_services/settings.py [hunk 2]
```diff
@@ -141,8 +141,8 @@
         'PORT': os.getenv('DB_COPY_PORT', '3306'),
         'OPTIONS': {
             "init_command": "SET default_storage_engine=InnoDB",
-        }
     }
+    },
```
Comment (james-monkeyshines) [inline production_services/settings.py:145]: This looks like a typo?

**Review by james-monkeyshines — APPROVED** [review]
Don't know enough to judge the Django code, but the interface on dev site looks good.
Maybe a couple of minor typos in ancillary files?

File: ensembl_production/templates/inc/headers.inc.html (hunk 1 above)
Comment (marcoooo) [reply ensembl_production/templates/inc/headers.inc.html:11]: Yes indeed.

File: production_services/settings.py (hunk 2 above)
Comment (marcoooo) [reply production_services/settings.py:145]: Nope just a reformat

**Pull request merged**